### ISSUE TITLE
Bulk operations fail after updating to 2.6.15

### ISSUE THREAD

**Issue by roman-fidalia** [zzzprojects/EntityFramework-Extensions#276]
### Description
Bulk operations fail (found with `BulkDelete`, tested with `BulkInsert` as well) after updating from `v2.6.11` to `v2.6.15`

### Exception
```
System.ArgumentNullException: Value cannot be null.
Parameter name: accessor
   at Microsoft.EntityFrameworkCore.Utilities.Check.NotNull[T](T value, String parameterName)
   at Microsoft.EntityFrameworkCore.Infrastructure.AccessorExtensions.GetInfrastructure[T](IInfrastructure`1 accessor)
   at .(DbContext , List`1 ,  , IEntityType )
   at .`1.( )
   at .Read()
   at .()
   at System.Data.SqlClient.SqlBulkCopy.ReadFromRowSource()
   at System.Data.SqlClient.SqlBulkCopy.ReadFromRowSourceAsync(CancellationToken cts)
   at System.Data.SqlClient.SqlBulkCopy.WriteToServerInternalAsync(CancellationToken ctoken)
   at System.Data.SqlClient.SqlBulkCopy.WriteRowSourceToServerAsync(Int32 columnCount, CancellationToken ctoken)
   at System.Data.SqlClient.SqlBulkCopy.WriteToServer(IDataReader reader)
   at .( , SqlConnection , SqlTransaction , Int32 )
   at .( )
   at .Execute(List`1 actions)
   at Z.BulkOperations.BulkOperation.Execute()
   at .[](DbContext this, IEnumerable`1 , Action`1 )
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
--- End of stack trace from previous location where exception was thrown ---
   at DbContextExtensions.BulkDeleteAsync[T](DbContext this, IEnumerable`1 entities, Action`1 options, CancellationToken cancellationToken)
```

### Further technical details
- EF version: `2.2.6`
- EF Extensions version: `v2.6.15`
- Database Provider: SQL

**Comment by JonathanMagnan** [comment]
Hello @roman-fidalia ,

Thank you for reporting.

We will try it on our side.

Best Regards,

Jonathan

**Comment by JonathanMagnan** [comment]
Hello @roman-fidalia ,

Could you try the v2.6.16,

We have found an issue that could be exactly the one you reported.

**Comment by roman-fidalia** [comment]
Will try as soon as the version is available from nuget.

**Comment by roman-fidalia** [comment]
Problem seems to be fixed in `v2.6.16`. Thanks.
